Week-3/Assignment 3.4/index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Assignment 3.4</title>
+</head>
+
+<body>
+    <script>
+
+        function createStack() {
+            const items = [];
+
+            return {
+                push(item) {
+                    items.push(item);
```

Comment:
Do this without the pre-defined array functions i.e don't use `items.push()`
It can be items  = [...items, item]